### PR TITLE
Create Rule “rules-to-better-automation/rule”

### PR DESCRIPTION
**Pull request by JackDuan123**
**Tip: Use [SSW Rule Writer GPT](https://chat.openai.com/g/g-cOvrRzEnU-ssw-rules-writer) for help with writing rules 🤖**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️ My random chat with seth

> 2. What was changed?

✏️ Adding a rule called rules to better automation

> 3. Did you do pair or mob programming (list names)?

✏️ No

### CODE REVIEW

**Review by danielmackay — CHANGES_REQUESTED** [review]
In general, this rule does not seem super clear to me.  I'd like to see a much clearer description of what the problem is and how your rule solves that.

File: rules/rules-to-better-automation/rule.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+type: rule
+title: Rules to better automation
+uri: rules-to-better-automation
+authors:
+  - title: Jack
+created: 2023-12-07T05:38:25.785Z
+guid: 9557813f-a26f-4f42-a420-6031f0f5f78f
+---
+A social group's success often depends on its conventions—shared norms and agreements that guide communication, decision-making, and unity. Strong conventions help navigate challenges, resolve conflicts, and maintain a positive atmosphere, shaping the group's overall success. It's crucial to note that while automation introduces efficiency, it should not disrupt these conventions or alter the fundamental ways people interact and collaborate. Automation should complement and enhance established norms rather than break them, ensuring a seamless integration that preserves the group's cohesiveness.
```
Comment (sethdailyssw) [inline rules/rules-to-better-automation/rule.md:10]: We do this all the time - we introduce automation that changes the way people work in the short term but gives efficiency benefits in the long term once people adjust.

If you want to make this rule you need to define 'convention' and at which points it's acceptable and unacceptable to break conventions.

It needs to be specific i.e. make it 'followable'

**Review by sethdailyssw — CHANGES_REQUESTED** [review]
We do this all the time - we introduce automation that changes the way people work in the short term but gives efficiency benefits in the long term once people adjust.

If you want to make this rule you need to define 'convention' and at which points it's acceptable and unacceptable to break conventions.

It needs to be specific i.e. make it 'followable'

**Review by sethdaily — CHANGES_REQUESTED** [review]
@JackDuan123 I don't think this rule is clear or specific

1. Improve it, or delete the branch

**Comment by danielmackay** [comment]
Closing this pull request due to lack of activity.  

@JackDuan123 - if this is is a change you really want to make, please update your branch and create another PR.

**Pull request closed without merging**